addons/web/static/src/views/view_button/view_button_hook.js

```diff
@@ -85,6 +88,7 @@ export function useViewButtons(model, ref, options = {}) {
                     error = _e;
                     await doActionParams.onClose();
                 }
+                await afterExecuteAction(clickParams);
```

Comment:
Do you want to execute afterExecuteAction if the doAction throw an error ?

Reply:
I think it's better to call it, there is one use case that uses it as cleanup.
Maybe we could pass the error? Or a boolean if it failed.
There is also the fact that if the action opens a dialog this might be called before the dialog closes, which might not make sense.